api/src/models/api/nova/ISlotResponse.ts

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line import/no-unresolved
+import { SlotCommitment } from "@iota/sdk-nova";
+
+export interface ISlotResponse {
```

Comment:
Responses usually extend the `IResponse` interface on client/api that adds the `error` field you can use to handle not found etc...